packages/charts/src/components/ScatterChart/ScatterChart.tsx

```diff
@@ -219,7 +225,11 @@ const ScatterChart: FC<ScatterChartProps> = forwardRef((props: ScatterChartProps
         <YAxis
           // eslint-disable-next-line @typescript-eslint/ban-ts-comment
           // @ts-ignore
-          label={yMeasure?.label ? { value: yMeasure?.label, angle: -90, position: 'insideLeft' } : false}
+          label={
+            yMeasure?.label
+              ? { value: yMeasure?.label, angle: -90, position: useIsRTL() ? 'insideRight' : 'insideLeft' }
```

Comment:
please don't call hooks inside props. extract it to `const isRTL = useIsRTL()` somewhere before the return and just use the value here.